applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -296,9 +296,10 @@ static void unicast_client_location_cb(struct bt_conn *conn, enum bt_audio_dir d
 	int ret;
 
 	if (dir == BT_AUDIO_DIR_SINK) {
-		if (loc == BT_AUDIO_LOCATION_FRONT_LEFT) {
+		if (loc & BT_AUDIO_LOCATION_FRONT_LEFT || loc & BT_AUDIO_LOCATION_SIDE_LEFT) {
```

Comment:
Is there a specific reason for adding 'side'?

Reply:
Couple products from market is using similar setting.